python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -389,6 +392,7 @@ class SensorResult(
                 "asset_events",
                 List[Union[AssetObservation, AssetMaterialization, AssetCheckEvaluation]],
             ),
+            ("automation_condition_evaluations", Optional[Sequence[AutomationConditionEvaluation]]),
```

Comment:
we don't really need to worry about size here, right, since these objects don't travel over a serdes boundary

Reply:
oh, wait, they do get serdesed in the other object. Do we need to worry about size here?

Reply:
They would travel over the serdes boundary for the intended use case actually -- the UserCodeAutomationConditionSensor evaluates on the user code server and returns a regular-old SensorResult (which has automation_condition_evaluations set).

However, it does do the "sensible" thing and only returns the list of evaluations for which something has changed, which is generally not a ton. What constraints are we working with in terms of size? I'd imagine the cursor would almost always be larger than the list of evaluations here (other than like the first tick, where all the evaluations happen from scratch)

Reply:
5 MB gzipped would be the typical limit here